greynoise/plugin.spec.yaml

```
@@ -0,0 +1,509 @@
+---
+
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: greynoise
+title: GreyNoise
+vendor: greynoise
+support: community
```

We can set this one to partner since it's contributed by the vendor

```suggestion
support: partner
```